es6.js

```diff
@@ -0,0 +1,59 @@
+class BuilderES6 {
+    constructor(integer) {
+        this.integer = integer || 0;
+    }
+}
+
+class IntBuilder extends BuilderES6 {
+    plus(...n) { 
+        this.integer = [...arguments, this.integer].reduce((accum, curr) => accum + curr, 0);
```

Comment:
`arguments` should be avoided. Instead you can use `...n` to loop over items that you already did but not used here.